app/assets/master.css

```diff
@@ -0,0 +1,73 @@
+html, body, p, th, td, li, dd, dt, pre {
+ font: 1em Verdana, Arial, Helvetica, sans-serif;
+ }
+
+div.login_bar {
+ z-index: 99999;
+ float:right; font-size:0.9em; border:1px;
+ line-height: 50px;	//size of the google+ user image
+ vertical-align:top;
+}
+
+.error { color:#FF0000; }
```

Comment:
We may want to add a linting step to the build for css as the different styles are being used - would be great to standardise on a single, common one. For example, selectors and rules should be consistently new-lined.